### DISCUSSION TITLE
pinia doesn't show state properly when useAsyncState has been used

### DISCUSSION THREAD

**Discussion by aslan0402** [vuejs/pinia#2668] [Help and Questions]
### Reproduction

playground doesn't connect vuecore properly....

### Steps to reproduce the bug

Install vue3, pinia, @vueuse/core

```
import { useAsyncState } from '@vueuse/core';
import { defineStore } from 'pinia';

const useSomeStore = defineStore('someStore', () => {
	const claimDeatailsStore = useClaimDetailsStore();

	const someData = useAsyncState(
		args => {
			return getSomeData(args);
		},
		[],
		{
			immediate: false,
			shallow: false
		}
	);

	return { someData };
});
```

Check VueDevTool pinia section. There will be empty space.

### Expected behavior

Pinia has to show data properly

### Actual behavior

Pinia doesn't show state inside vuedevtool if useAsyncState has been used from '@vueuse/core' lib.

**Comment by kevinramharak** [comment]
@aslan0402 You need to setup the import maps properly:
https://play.pinia.vuejs.org/#__DEV__eNqVVEtP4zAQ/itWLm2l1mkBsVIpqCzisHvYRQtHX0IyLWbjh2wn7arqf9+xnaShsKirHBzP45tvHp5dcqs1rStI5snC5oZrRyy4SpMyk+trljjLkhsmudDKOLIjlYVHpwyQPVkZJciAprmqpANDnR0wyWSupEWQYHTd2Q9HTC7SGAHx8OJA6DJzgDdCFgWvbxqgOdntoj9tJGS/X6TewmP0/JJxEolNRKbpq1US09h5PNYokD3CeYmXaS555iUseXFO23ma5oVExwJKXhsqwaVSizTYLc/ojH7BsNZFAQUrJs9GbSwm+4rI4xZ3iQVMC6idUqWdZJqfEOOdz/KSXtJZWvLnFAOlXBawfRsGPSYFiFPQW9PllM4uMAuPWp83oOIdeewSttHAibxb8+UM4Wd0+imufckMFP+BHB0+wvbQeyb32Hdncc5WfH3U9VwJzUswP7XjOIdvup+Vpdp8DzJnKuiI5i+Q//5A/mq3kfWDAWx5jdXpdC4za3BRff/4A7b43ymFKqqyqeU/lL/AqrLyHKPZ10oWSLtnF9h+C0PM5frJ3m8dSNsm5YmGagT7MBp3n6R+oHtOz3tVPDxdLGH/jd/aPzJ/dPjMuofeb/zgqrcSClhxebQVwoPxVkxmHoqsKpl7XgR53PmoIcJwFDlmm4w7ImFDHtCdWxhiyVVZ4wa58fvoiQtQlWulY3I2nU5HI8QnxOC6MpJcnOEN02IStoFY3EPdvrru8xwOwn4ZjAkywBBIAj8XXdqlE5bXoRBDb8HcwQEvzDXRj7LyiaPSd8ef0+bsvLgQUHAE7UYuynH0fdvnZJWVFhscQPwREPFswu1akvP2h9rYLF+E0VWy/wsXfQLX

Can you use that to create a proper reproduction?
